thingif/src/main/java/com/kii/thingif/command/Action.java

```diff
@@ -30,5 +24,12 @@
  *
  */
 public interface Action {
+    /**
```

Comment:
API document says ` **Note** the return value must be neither null nor empty.`.

If we make `Action` as abstract class, we can check it inside of this class.

However, Almost all of application developers might implement `getActionName()` in following style:

```java

public String getActionName() {
    return "turnPower"
}
```

So my suggestion might be thinking too much :)
Please forget this suggestion.